admin/class-doppler-for-woocommerce-admin.php

```diff
@@ -327,6 +327,36 @@ private function find_list_by_name($list_name, $lists) {
 	    return ($resp!=='null')? key($resp) : false;
 	}
 
+	/**
+	 * Get the WooCommerce customer's fields.
+	 */
+	public function sync_all_buyers() {

```

Comment:
```suggestion
	/**
	 * Set buyers and contacts last synch value to 0.
	 */
	public function reset_buyers_and_contacts_last_synch() {

```
Que les parece? creo que seria mas preciso en cuanto a lo que realmente hace la funcion.